source/common/router/router.cc

```diff
@@ -1216,6 +1216,7 @@ bool Filter::setupRetry() {
   // this filter which will make this a non-issue. The implementation of supporting retry in cases
   // where the request is not complete is more complicated so we will start with this for now.
   if (!downstream_end_stream_) {
+    config_.stats_.rq_retry_skipped_after_request_completed_.inc();
```

Comment:
Isn't this checking that we've fully received the request? Would it make more sense word this as `rq_retry_skipped_request_not_complete` or something to that effect?

Reply:
Yeah. That makes sense